### PR TITLE
Slim down Sanity moving some tests to Tier2 (execution time > 15 mins)

### PR DESCRIPTION
**Pull request by jgarciao**
After agreement on Google Chat, this PR is moving some tests from Sanity/Tier1 to Tier2 because they need more than 15 mins to execute

### CODE REVIEW

**Review by lugi0 — APPROVED** [review]

File: tests/Tests/400__ods_dashboard/400__ods_dashboard.robot [hunk 1]
```diff
@@ -163,8 +163,9 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
 
 Verify Notifications Appears When Notebook Builds Finish And Atleast One Failed
     [Documentation]    Verifies that Notifications are shown when Notebook Builds are finished and atleast one fails
-    [Tags]    Sanity
+    [Tags]    Tier2
```
Comment (bdattoma) [inline tests/Tests/400__ods_dashboard/400__ods_dashboard.robot:166]: should be Tier3 for long running tests

File: tests/Tests/500__jupyterhub/image-iteration.robot [hunk 2]
```diff
@@ -32,13 +32,15 @@ Iterative Testing Classifiers
   END
 
 Iterative Testing Clustering
-  [Tags]  Sanity  POLARION-ID-Clustering
+  [Tags]  Tier2  POLARION-ID-Clustering
```
Comment (bdattoma) [inline tests/Tests/500__jupyterhub/image-iteration.robot:35]: should be Tier3 for long running tests

**Review by bdattoma — APPROVED** [review]
approving it. But I think we will need to review Tier2/Tier3, because the test cases which last long time should go to Tier3. But currently we are placing them sometimes to Tier2 and sometimes to Tier3.

File: tests/Tests/500__jupyterhub/image-iteration.robot (hunk 2 above)
Comment (jgarciao) [reply tests/Tests/500__jupyterhub/image-iteration.robot:35]: As agreed in the chat, we'll merge this PR with Tier2 in order to release ods-ci 1.13, but we'll discuss in the automation meeting if Tier3 is more appropriate

**Comment by jgarciao** [comment]
As agreed in the chat, we'll merge this PR with Tier2 in order to release ods-ci 1.13, but we'll discuss in the automation meeting if Tier3 is more appropriate

**Pull request merged**